### PR TITLE
Release candidate v2.3.1

### PR DESCRIPTION
**Pull request by adamrtalbot**
- Add support for alternative container registry
- Update to NF-Core template 2.8

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/scrnaseq/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/scrnaseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,10 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v2.4.0dev - [date]
```
Comment (maxulysse) [inline CHANGELOG.md:6]: changed in https://github.com/nf-core/scrnaseq/pull/227

File: nextflow.config [hunk 2]
```diff
@@ -223,7 +257,7 @@ manifest {
     description     = """Pipeline for processing 10x Genomics single cell rnaseq data"""
     mainScript      = 'main.nf'
     nextflowVersion = '!>=22.10.1'
-    version         = '2.3.0'
+    version         = '2.4.0dev'
```
Comment (maxulysse) [inline nextflow.config:260]: changed in https://github.com/nf-core/scrnaseq/pull/227

**Review by maxulysse — APPROVED** [review]
LGTM

**Review by maxulysse — APPROVED** [review]

**Review by maxulysse — APPROVED** [review]

**Review by maxulysse — APPROVED** [review]

File: lib/WorkflowMain.groovy [hunk 3]
```diff
@@ -20,8 +22,8 @@ class WorkflowMain {
     //
     // Generate help string
     //
-    public static String help(workflow, params, log) {
-        def command = "nextflow run ${workflow.manifest.name} --input samplesheet.csv --fasta human.fasta --gtf human.gtf -profile docker"
+    public static String help(workflow, params) {
+        def command = "nextflow run ${workflow.manifest.name} --input samplesheet.csv--fasta human.fasta --gtf human.gtf -profile docker"
```
Comment (ejseqera) [inline lib/WorkflowMain.groovy:26]:
```suggestion
        def command = "nextflow run ${workflow.manifest.name} --input samplesheet.csv --fasta human.fasta --gtf human.gtf -profile docker"
```

**Review by ejseqera — APPROVED** [review]
🚀

**Pull request merged**